plugins/colordialog/dialogs/colordialog.js

```diff
@@ -22,6 +22,14 @@ CKEDITOR.dialog.add( 'colordialog', function( editor ) {
 		html: '&nbsp;'
```

Comment:
Color dialog hasn't got any unit test earlier.
This change:
```JavaScript
if ( focused ) {
	focused.removeClass( focusedColorLightCls );
	focused.removeClass( focusedColorDarkCls );
}
```
Allows on making nice unit test which set up values but doesn't touch focused colour. So test don't throw up errors when only value exists without focusing it.